### PR TITLE
change userId from string to interface

### PR DESCRIPTION
**Pull request by stackerzzq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ import (
 	"os"
 	"time"
 
-	"github.com/appleboy/gin-jwt"
 	"github.com/gin-gonic/gin"
+	"github.com/stackerzzq/gin-jwt"
```
Comment (appleboy) [inline README.md]: revert this.

File: vendor/vendor.json [hunk 2]
```diff
@@ -74,6 +74,12 @@
 			"revision": "792786c7400a136282c1664665ae0a8db921c6c2",
 			"revisionTime": "2016-01-10T10:55:54Z"
 		},
+		{
```
Comment (appleboy) [inline vendor/vendor.json]: revert this changes

**Comment by appleboy** [comment]
@stackerzzq Any updates?

**Comment by stackerzzq** [comment]
Already fixed, please review again.

**Comment by appleboy** [comment]
See #118
